js/index.js

```diff
@@ -0,0 +1,38 @@
+console.clear();
+
+const showAnswerButton1 = document.querySelector('[data-js="show-answer1"]');
+const answer1 = document.querySelector('[data-js="answer1"]');
+const showAnswerButton2 = document.querySelector('[data-js="show-answer2"]');
+const answer2 = document.querySelector('[data-js="answer2"]');
+const showAnswerButton3 = document.querySelector('[data-js="show-answer3"]');
+const answer3 = document.querySelector('[data-js="answer3"]');
+
+showAnswerButton1.addEventListener("click", () => {
+  if (showAnswerButton1.textContent == "Show Answer") {
```

Comment:
I wonder whether the if-statement is neccessary here or maybe can even be left out? But i'm not sure